frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -34,15 +34,17 @@ for (const mode of modeList) {
 export default class FeedbackCodeView extends Vue {
   @Prop() language!: string;
   @Prop() value!: string;
-  @Prop({ default: () => new Map<number, FeedbackClass>() }) feedbackMap!: Map<
-    number,
-    ArenaCourseFeedback
-  >;
+  @Prop({ default: () => new Map<number, typeof FeedbackClass>() })
+  feedbackMap!: Map<number, Feedback>;
   @Ref('cm-editor') private readonly cmEditor!: HTMLTextAreaElement;
 
   T = T;
   mode = languageModeMap[this.language] ?? languageModeMap['cpp17-gcc'];
-  numberOfComments = 0;
+  mapChangeTracker = 0;
```

Comment:
```suggestion
  mapChangeTracker = 1;
```

esto resulta que es importante para que el workaround jamás regrese "falso".

Reply:
Listo